lib/v8-log-to-ticks.js

```diff
@@ -3,11 +3,32 @@
 const { spawn } = require('child_process')
 const { parse } = require('jsonstream2')
 const { extname } = require('path')
+const { promisify } = require('util')
 const through = require('through2')
 const fs = require('fs')
 const pump = require('pump')
 
-module.exports = v8LogToTicks
+const readFile = promisify(fs.readFile)
+const writeFile = promisify(fs.writeFile)
+
+module.exports = process
+
+function process (isolateLogPath, node) {
+  return filterLinesThatAreTooLong(isolateLogPath).then((filtered) => {
```

Comment:
I suggest to only execute this when it's necessary (by checking the Node.js version).